code/style.css

```diff
@@ -1,9 +1,239 @@
-/* 
-If you can see a peach background, the css file is correctly linked. 
-Feel free to change or remove this styling! 
-We want you focus A LOT on the styling for this project. 
-*/
-
+* {
+  box-sizing: border-box;
+}
+  
 body {
-  background-color: peachpuff;
+  font-family: 'Roboto', Verdana, Calibri, sans-serif;
+  font-weight: 300;
+  font-size: 16px;
+  margin: 0;
+}
+
+.background {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background-image: url(./images/gruppe.jpeg);
+  background-position: center;
+  background-size: cover;
+  height: 650px;
+}
+
+h1 {
+  font-family: 'Segoe UI', Verdana, Calibri, sans-serif;
+  font-size: 50px;
+  color: rgb(22, 22, 75);
+}
+
+nav {
+  line-height: 2;
+}
+
+.arrow {
+  border: solid rgb(22, 22, 75);
+  border-width: 0 1px 1px 0;
+  display: inline-block;
+  margin: 0 7px;
+  padding: 3px;
+  transform: rotate(45deg);
+  -webkit-transform: rotate(45deg);
+}
+
+.drop {
+  border: none;
+}
+.dropdown-content {
+  display: none;
+  background-color: rgb(220, 220, 220, 0.5);
+  text-align: center;
+  width: 90px;
+}
+.dropdown:hover .dropdown-content {
+  display: block;
+}
+
+header {
+  text-align: right;
+  padding: 30px 50px 0 0;
+  background-image: url(./images/gruppe.jpeg);
+  background-size: cover;
+  height: 80px;
+}
+
+a.black:link, a.black:visited {
+  color: black;
+}
+
+a.blue:link, a.blue:visited {
+  font-family: 'Segoe UI', Verdana, Calibri, sans-serif;
+  font-size: 18px;
+  color: rgb(22, 22, 75);
+  text-decoration: none;
+}
+
+a.none:link, a.none:visited {
+  color: black;
+  text-decoration: none;
+}
+
+span {
+  text-align: center;
+  font-size: 18px;
+}
+
+h3 {
+  font-size: 22px;
+  font-weight: 400;
+}
+
+.gruppen, .kontakt, .form {
+  display: flex;
+  flex-direction: column;
+}
+
+.gruppen img {
+  max-height: 500px;
+  object-fit: cover;
+  margin: 10px;
+}
+
+.about {
+  align-self: center;
+  padding: 0 35px;
+}
+
+.coach {
+  margin: 60px 0 25px 0;
+  display: flex;
+}
+.coach img {
+  margin-left: 25px;
+  object-fit: cover;
+  border-radius: 50%;
+}
+
+h2 {
+  margin-top: 40px;
+  text-align: center;
 }
+
+.kontakt {
+  border-bottom: solid;
+}
+
+.name {
+  text-align: center;
+  line-height: 2;
+  margin-bottom: 25px;
+}
+
+h4, legend {
+  font-weight: 500;
+}
+
+.anmeldung {
+  margin: 15px 8%;
+}
+
+form {
+  margin: 35px 0;
+  padding: 0 25px;
+}
+
+.first-row, .second-row {
+  width: 100%;
+}
+
+fieldset {
+  margin-bottom: 15px;
+  padding-top: 25px;
+  border: none;
+}
+
+input[type=text], input[type=date], input[type=tel], input[type=email], select {
+  width: 100%;
+  border: none;
+  border-radius: 4px;
+  padding: 3px;
```

Comment:
I think some more padding would look nice!

Reply:
Also - what do you think about changing the font size of the placeholder to the same as the labels have?